core/src/test/java/org/elasticsearch/bootstrap/BootstrapCheckTests.java

```diff
@@ -197,4 +199,15 @@ public void testEnforceLimits() {
         assertTrue(BootstrapCheck.enforceLimits(settings));
     }
 
+    public void testMinMasterNodes() {
+        BootstrapCheck.Check check = new BootstrapCheck.MinMasterNodesCheck();
+        assertTrue(check.check(Settings.EMPTY));
+        assertFalse(check.check(
+            Settings.builder().put(ElectMasterService.DISCOVERY_ZEN_MINIMUM_MASTER_NODES_SETTING.getKey(), 1).build()
+        ));
+        expectThrows(RuntimeException.class, () ->
+            BootstrapCheck.check(
```

Comment:
> MinMasterNodeCheck does the right thing based on whether or not discovery.zen.minimum_master_nodes is set 

The problem is that now that we moved the settings check out of MinMasterNodeCheck and into the code that calls it's constructor, I need to make sure it's created correctly and does the right thing. I changed the test to create the default checks and pass them to the check methods.